addons/mail/static/src/messaging/entity_errors.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('mail.messaging.entityErrors', function (require) {
+'use strict';
+
+class EntityDeletedError extends Error {
+
+    /**
+     * @override
+     * @param {string} entityLocalId
+     */
+    constructor(entityLocalId) {
+        super(...arguments);
+        this.deletedEntityLocalId = entityLocalId;
```

Comment:
It is strange to couple this extra key (if needed) with the message itself (passed to super).